aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -539,7 +554,7 @@ private int replayToken()
         if (NULL_VALUE != replayToken || !ChannelUri.parse(replicationChannel).hasControlModeResponse())
         {
             workCount++;
-            state(State.GET_ARCHIVE_PROXY);
+            state(State.GET_ARCHIVE_PROXY, "replay token"); // TODO
```

Comment:
Outstanding TODO.

Reply:
Use an empty string here. This is a technical state transition that is only happening when response channels feature is used during replication.